mobile/library/jni/jni_utility.cc

```diff
@@ -426,5 +426,103 @@ absl::flat_hash_map<std::string, std::string> javaMapToCppMap(JniHelper& jni_hel
   return cpp_map;
 }
 
+LocalRefUniquePtr<jobject>
+cppHeadersToJavaHeaders(JniHelper& jni_helper,
```

Comment:
do we have plans to use this on the response path, or is this test only code?  if it's test only code we could move it to the test jni utils area for clarity

Reply:
At the moment no, but since it's just a utility function, I think it's fine to keep it in the `JniUtility` given that the utility will contain bidirectional conversion, which IMO is nice to have.